HW/iris.py

```diff
@@ -1,75 +1,142 @@
+#!/usr/bin/env python3
+
+import collections
+import os
+import sys
+
 import numpy as np
 import pandas as pd
 import plotly.express as px
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.metrics import accuracy_score, classification_report, confusion_matrix
```

Comment:
when I run pre-commit hooks on this file isort changes this line. But barely. It might just be an error on my isort version or something.